pybossa/view/projects.py

```diff
@@ -1150,7 +1150,8 @@ def export(short_name, task_id):
     if task:
         taskruns = task_repo.filter_task_runs_by(task_id=task_id, project_id=project.id)
         taskruns_info = [tr.dictize() for tr in taskruns]
-        gold_answers = task.gold_answers if task.calibration and task.gold_answers else {}
+        can_know_task_is_gold = current_user.subadmin or current_user.admin
```

Comment:
This looks good. Just something to consider.

I'm wondering if it could be more readable to name the variable `is_admin_subadmin` rather than `can_know_task_is_gold`, which is slightly verbose and maybe overly specific? I think either way is fine, and maybe we want to be this specific here. It's just something to consider. Thank you. :)

The subsequent logic would then be:
```python
if is_admin_subadmin:
   # Show gold task details.
```

```python
{% if is_admin_subadmin %}
   <div>Show gold details.</div>
{% endif %}